src/main/java/xyz/thepathfinder/android/ModelRegistry.java

```diff
@@ -13,6 +14,9 @@
 class ModelRegistry {
 
     private static final Logger logger = Logger.getLogger(ModelRegistry.class.getName());
+    static {
+        logger.setLevel(Level.INFO);
```

Comment:
Also a developer using our SDK should not need to know the names and functions of internal classes